python/torch_mlir/extras/fx_importer.py

```diff
@@ -888,6 +888,20 @@ def node_val_to_type(self, node: torch_fx.Node, *, mutable: bool = False) -> IrT
             tensor_meta = node.meta.get("tensor_meta")
             val = node.meta.get("val")
             sparsity = node.meta.get("sparsity", None)
+            # Some nodes returns a list, like torch.ops.aten.unbind.int
+            if isinstance(tensor_meta, List) or isinstance(val, List):
```

Comment:
I am pretty sure that the second clause should be `isinstance(val, list)`: `List` is a type descriptor that is never instantiated. `list` is the runtime type.